firebase-ai/CHANGELOG.md

```diff
@@ -62,13 +66,5 @@
     free tier offering.
   - Using the Firebase AI SDK with the Vertex AI Gemini API is still generally available (GA).
 
-If you're using the old `firebase-vertexai`, we recommend
```

Comment:
was this removal intended?

Reply:
Unclear why it happened, but I restored it.